pkg/repository/v1/sqlcv1/workflows.sql

```
@@ -208,6 +208,17 @@ INSERT INTO "WorkflowTriggerEventRef" (
 ) RETURNING *;
 
 -- name: CreateWorkflowTriggerCronRef :one
+WITH previous_trigger AS (
+    SELECT "enabled"
+    FROM "WorkflowTriggerCronRef" c
+    JOIN "WorkflowTriggers" t ON t."id" = c."parentId"
+    JOIN "WorkflowVersion" wv ON wv."id" = t."workflowVersionId"
+    WHERE
+        wv."id" = sqlc.narg('oldWorkflowVersionId')::uuid
+        AND "cron" = @cronTrigger::text
+        AND "name" = sqlc.narg('name')::text
+)
```

@abelanger5 verifying this is okay to do - the goal here is to move the `enabled` flag onto the new cron ref from the old one. main risk seems to be the possibility of this returning multiple rows, which I don't think should happen but I'm not sure

---

Yeah shouldn't be possible given the unique constraint we have on the cron - but perhaps you can return `LIMIT 1` to be safe?

---

I was nervous that `LIMIT 1` would mask an error - I'd sorta prefer it failed loudly 😅